src/playground/components/RuleItem.js

```diff
@@ -1,16 +1,5 @@
-import React, { useRef, useEffect } from "react";
+import React from "react";
 
 export default function RuleItem({ children }) {
-    const ref = useRef();
-
-    useEffect(() => {
-        ref.current.scrollIntoView({ behavior: "smooth", block: "end" });
-        ref.current.children[1].focus();
-    }, []);
-
-    return (
-        <li ref={ref} className="config__added-rules__item">
-            {children}
-        </li>
-    );
+    return <li className="config__added-rules__item">{children}</li>;
```

Comment:
Just wondering if we'd need a separate file for this one line? This could be inside configuration itself? We don't want this to be a component? WDYT?

Reply:
Yes, we can move it to `Configuration`. It doesn't make sense to keep it in a separate file.